assets/conf.d/ephemeral.yaml

```diff
@@ -0,0 +1,3 @@
+extensions:
```

Comment:
Isn't this the wrong file path? It's in conf.d instead of conf.d-available

Reply:
This is just the source path in the repository. I can change it though so it's less confusing.